ci-operator/step-registry/optional-operators/subscribe/optional-operators-subscribe-commands.sh

```diff
@@ -105,6 +105,27 @@ EOF
 
 echo "CatalogSource name is \"$CATSRC\""
 
+IS_CATSRC_CREATED=false
+# Wait for 10 minutes until the Catalog source state is 'READY'
+for i in $(seq 1 120); do
+    CATSRC_STATE=$(oc get catalogsources/"$CATSRC" -n "$OO_INSTALL_NAMESPACE" -o jsonpath='{.status.connectionState.lastObservedState}')
+    echo $CATSRC_STATE
+    if [ "$CATSRC_STATE" = "READY" ] ; then
+	echo "Catalogsource created successfully after waiting $((5*i)) seconds"
+        echo "current state of cataloguesource is \"$CATSRC\""
+        IS_CATSRC_CREATED=true
+        break
+    fi
+    sleep 5
+done
+
+if [ $IS_CATSRC_CREATED = false ] ; then
+    echo "Timed out waiting for the catalog source $CATSRC to become ready after 10 minutes."
+    echo "Catalogsource state at timeout is \"$CATSRC_STATE\""
+    echo "Catalogsource image used is \"$OO_INDEX\""
+    exit 2
```

Comment:
Why do we exit with return code 2?
All other places use either 1 or 0
* `exit 1` is used 3 times
* `exit 0` is used once

Reply:
@lslebodn It should be exit 1 
It was a typo